File: Backend/Domain/Models/LoanModel.cs
using Domain.Enums;

namespace Domain.Models
{
    public class LoanModel
    {
        public Guid? Id { get; set; }
        public Guid? ClientBankAccountId { get; set; }
        public LoanType LoansTypesId { get; set; }
        public string ClientAccountNumber { get; set; } = default!;
        public string LoanAmount { get; set; } = default!;
        public string Income { get; set; } = default!;
        public EmploymentStatus EmploymentStatus { get; set; } = default!;
        public string? LoanPeriod { get; set; }
        public double? InterestRate { get; set; }
        public string? MonthlyPayment { get; set; }

    }
}

File: Frontend/src/interfaces/loan-model.ts
import { loanType } from "./LoanType";
import { employmentStatus } from "./employmentStatus";

export interface LoanModel {
    id:string | null;
    userId:string | null;
    loanAmount: string | null;
    interestRate:number;
    monthlyPayment:string|null;
    loanPeriod:string | null;
    income:string|null;
    employmentStatus:employmentStatus;
    clientBankAccountId: string |null;
    clientAccountNumber:string ;
    loansTypesId: loanType ;
    userName:string|null;
}